scio-examples/src/main/scala/com/spotify/scio/examples/extra/AnnoyExample.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.extra.annoy._
+import com.spotify.scio.values.SCollection
+/**
+ * sbt "project scio-examples" "runMain com.spotify.scio.examples.extra.AnnoyExample \
+ * --runner=DataflowRunner --tempLocation=gs://<bucket>/<path>"
+ */
+object AnnoyExample {
+
+  def main(cmdlineArgs: Array[String]): Unit = {
+    val (sc, args) = ContextAndArgs(cmdlineArgs)
+
+    val metric = Angular
+    val dim = 40
+    val nTrees = 10
+
+    // open annoy file from scio context
+    //val path = args("input")
```

Comment:
What is this comment for? if it's another example maybe there should be two examples?

Reply:
Alright - i see, maybe just have the `args` version here, and the rest could be documented in tests?

Reply:
It's meant to be another example, but the code for accessing the data loaded would be similar. Would it be better to put a switch in the command line arguments?

Reply:
I think it probably better to keep this as close to "real job" as possible, and real job would not create dummy data. I think i would have just one example but close to "real job".